coil-base/src/main/java/coil/memory/RequestService.kt

```diff
@@ -145,10 +147,10 @@ internal class RequestService {
         return transformations.isEmpty() || Transformation.VALID_CONFIGS.contains(bitmapConfig)
     }
 
-    private fun Request.isConfigValidForHardware(size: Size): Boolean {
+    private fun Request.isConfigValidForHardware(size: Size, target: Target?): Boolean {
```

Comment:
`target` is already part of the `Request` receiver so we don't need to pass it in.